ddtrace/contrib/django/middleware.py

```diff
@@ -116,6 +116,11 @@ def process_response(self, request, response):
         try:
             span = _get_req_span(request)
             if span:
+                if response.status_code < 500 and span.error:
```

Comment:
👍 it avoids useless calls if we don't have an error because it has been removed by something else